teste.py

```
@@ -0,0 +1 @@
+teste
```

Ruff (0.8.2)

Found useless expression. Either assign it to a variable or remove it.

(B018)